### PR TITLE
Created navbar for landing and added custom link action feature for button

### PR DESCRIPTION
**Pull request by ArielCalisaya**
>PR created for issue [#1620](https://github.com/4GeeksAcademy/About-4Geeks-Academy/issues/1620)

### Tested in [becas-partner](https://website-v2-git-feature-landingnavbar-geeksacademy.vercel.app/es/landing/becas-partner) page

### CODE REVIEW

File: src/assets/css/style.css [hunk 1]
```diff
@@ -10,6 +10,8 @@ html, body
     margin: 0px!important;
     padding: 0px!important;
     overflow-x: clip!important; 
+    scroll-behavior: smooth;
```
Comment (ArielCalisaya) [inline src/assets/css/style.css:13]: Added this style for `smoth` scroll when the link points to an `id`

File: src/data/landing/becas-partner.es.yml [hunk 2]
```diff
@@ -15,6 +15,11 @@ meta_info:
   utm_course: full-stack
   utm_location: madrid-spain
 
+navbar:
+  buttonText: "Solicitar"
+  # url accepts: relative(/apply), absolute(https://google.com) or id(#componentName)
+  url: "#apply_schollarship"
```
Comment (ArielCalisaya) [inline src/data/landing/becas-partner.es.yml:21]: The url works with links `relative`, `absolute` and `id`(name of the function in landing)

File: src/data/landing/becas-partner.es.yml [hunk 3]
```diff
@@ -15,6 +15,11 @@ meta_info:
   utm_course: full-stack
   utm_location: madrid-spain
 
+navbar:
+  buttonText: "Solicitar"
```
Comment (ArielCalisaya) [inline src/data/landing/becas-partner.es.yml:19]: `buttonText` is optional

File: src/new_components/Landing/index.js [hunk 4]
```diff
@@ -187,6 +187,7 @@ export const landingSections = {
     about4Geeks: ({session, data, pageContext, yml, index}) => {
         return(
             <About4Geeks 
+                id="about4Geeks"
```
Comment (ArielCalisaya) [inline src/new_components/Landing/index.js:190]: I have predefined all the id based on the component name.

**Review by ArielCalisaya — COMMENTED** [review]
>changes have been reviewed

**Review by alesanchezr — APPROVED** [review]

**Pull request merged**